tests/cli/open_package/test_export.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+
+import params
+import pytest
+from helpers import _clean, nofmt
+
+"""
+Test command "open-package --export"
+"""
+
+
+@pytest.mark.parametrize(
+    "pkg_uuid",
+    [
+        "0eaf289c-166d-4bd9-a4ba-dbf6bbc76ef1",  # A package with multiple footprints
+    ],
+)
```

Comment:
Parametrization is actually not needed here - like in the test above, you could just put the UUID into the test code.

But it's not a problem of course, so no need to change.